x-pack/winlogbeat/module/sysmon/test/testdata/sysmon-11-registry.evtx.golden.json

```diff
@@ -14,7 +14,7 @@
       "level": "information"
     },
     "process": {
-      "entity_id": "{5b522f6e-77ae-5eb1-2c03-000000000800}",
+      "entity_id": "{5B522F6E-77AE-5EB1-2C03-000000000800}",
```

Comment:
These were generated from Window 2012. Marc is going to regenerate from Win 2019 and these changes should go away.